docs/snapshotting/random-for-clones.md

```diff
@@ -41,6 +43,8 @@ and there’s also an input pool which gathers entropy from various sources
 available on the system, and is used to feed into or seed the other two
 components. A very detailed description is available [here][3].
 
+### Linux kernels from 4.8 until 5.17 (included)
```

Comment:
Would be nice to mention what happens with VmGenId device on older kernels.

Reply:
will do